functionaltests/src/test/java/specs/response/HealthResponseFixture.java

```diff
@@ -34,7 +37,7 @@ public MultiValueResult handleHealthResponse() throws Exception {
 
         JsonObject rootNode = parser.parse(responseString).getAsJsonObject();
         String bodyStatus = rootNode.get("status").getAsString();
-        Optional<JsonArray> tenantNames = ofNullable(rootNode.get("tenants")).filter(JsonElement::isJsonArray).map(JsonElement::getAsJsonArray);
+        Optional<Set<Map.Entry<String, JsonElement>>> tenantNames = ofNullable(rootNode.get("tenants")).map(e -> e.getAsJsonObject().entrySet());
```

Comment:
Why instead of `Optional<Set<Map.Entry<String, JsonElement>>>` not use just `Map<String, JsonElement>`?

1. I mean a Map is basically also a set of map entries, so why the complex structure?
2. Why `Optional`, why not just empty map instead of empty optional?

Reply:
befbfc6a1

Reply:
better talk about this f2f.